docs/example_compartmental_elongated_cells_tutorial.rst

```diff
@@ -0,0 +1,805 @@
+Building Simulation of Elongated Cells. Case Study in using Compartments, FPP Links, Curvature energy terms.
+------------------------------------------------------------------------------------------------------------
+
+The goal of this tutorial is to build a simulation of multiple elongated cells where each sell is composed of compartments.
+We want the cells to stay elongated throughout the course of the simulation. In this tutorial we want to show you how
+starting from a simulation involving a single cell you can scale up and build more sophisticated models
+
+
+Understanding Contact Energies - how to avoid pixelated cells
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now that we know about compartments and how they are handled by CC3D, let's see how we can  translate our knowledge
+into building a simple simulation that involves contact energies. In particular, we want to show you a common pitfall
+that you may encounter in your work and how to diagnose it.
+
+In fact this tutorial is an intro to a more sophisticated simulation where we will leverage Focal Point Plasticity Plugin
+and Curvature plugin.
+
+.. note::
+
+    Our intention is to teach you how you can start building complex simulation from grounds up
+    by starting with a single ce,, understand the behavior of the single cell under different set of parameters
+    and gradually adding complexity to your simulation. We strongly believe that in order to build robust and complex
+    simulations you firs must master simple cases and build confidence needed to bring your modeling skills
+    to the "next level". It is very much like playing the piano, in general it is advised to learn how to play
+    "Chopsticks" https://www.youtube.com/watch?v=JM5fjgiFrxg before attempting to play the "Flight of the Bumblebee" https://www.youtube.com/watch?v=M93qXQWaBdE
+
+Let's start. Our first simulation will involve single cell of type "Top" and two plugins - Volume and Contact.
+The goal is to make the cell look non-pixelized and do not disappear.
+
+The code for this simulation can be found in ``Demos/CompuCellPythonTutorial/ElongatedCellsTutorial/Tutorial_01``
```

Comment:
Should we have a link to the CompuCell repos, or should we just add a .`..note::` that demos can be opened from `File -> Demo Browser`?